compiler/src/main/scala/org/scalajs/core/compiler/JSTreeExtractors.scala

```diff
@@ -15,6 +15,13 @@ object JSTreeExtractors {
 
   object jse {
```

Comment:
We'll probably need to move this to IR, since #1265 will also benefit from it. But we can do this in a later commit.